content/sensu-go/5.10/reference/filters.md

```diff
@@ -385,6 +385,10 @@ For more information about event attributes, see the [event reference][28].
 `event.entity.system.platform_version` | string  | The entity’s operating system version
 `event.entity.user`                    | string  | Sensu [RBAC][25] username used by the agent entity
 
+### Filter Assets
```

Comment:
Up on line 80, there is already a filter assets heading. Would it make sense to combine these?